### ISSUE TITLE
tag_drop out (while fully updated as far as I can tell)

### ISSUE THREAD

**Issue by Merijeek** [d8ahazard/sd_dreambooth_extension#352]
tag_drop_out (while fully updated, as far as I can tell)
```

E:\NEWSD\stable-diffusion-webui>webui-user.bat
Already up to date.
venv "E:\NEWSD\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 0b5dcb3d7ce397ad38312dbfc70febe7bb42dcc3
Installing requirements for Web UI
Installing requirements for scikit_learn

Checking Dreambooth requirements...
Checking/upgrading existing torch/torchvision installation
Checking torch and torchvision versions
Dreambooth revision is d49c1b3ab7d7463f5c1a91ebd3061c0b16638ac4
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.8.1 installed.
[+] transformers version 4.21.0 installed.
[+] torch version 1.12.1+cu116 installed.
[+] torchvision version 0.13.1+cu116 installed.
[ ] xformers version 0.0.14.dev0 installed.

Launching Web UI with arguments:
Patching transformers to fix kwargs errors.
Dreambooth API layer loaded
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading weights [74f4c61c] from E:\NEWSD\stable-diffusion-webui\models\Stable-diffusion\redshift-diffusion-v1.ckpt
Applying cross attention optimization (Doggettx).
Model loaded.
Loaded a total of 4 textual inversion embeddings.
Embeddings: liiv-v3-neg, liiv-v3, liv4-neg, liv4
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

My attempt to train in Dreambooth:

```

To create a public link, set `share=True` in `launch()`.
Starting Dreambooth training...
 Cleanup completed.
 Allocated: 0.0GB
 Reserved: 0.0GB

 Allocated 0.0/2.0GB
 Reserved: 0.0/2.0GB

Replace CrossAttention.forward to use default
 Cleanup completed.
 Allocated: 0.0GB
 Reserved: 0.0GB

 Loaded model.
 Allocated: 0.0GB
 Reserved: 0.0GB


===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
For effortless bug reporting copy-paste your error into this form: https://docs.google.com/forms/d/e/1FAIpQLScPB8emS3Thkp66nvqwmjTEgxp8Y9ufuWTzFyr9kJ5AoI47dQ/viewform?usp=sf_link
================================================================================
CUDA SETUP: Loading binary E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cuda116.dll...
 Cleanup completed.
 Allocated: 0.2GB
 Reserved: 0.2GB

 Scheduler, EMA Loaded.
 Allocated: 3.8GB
 Reserved: 3.9GB

***** Running training *****
  Num examples = 31
  Num batches each epoch = 31
  Num Epochs = 33
  Instantaneous batch size per device = 1
  Total train batch size (w. parallel, distributed & accumulation) = 1
  Gradient Accumulation steps = 1
  Total optimization steps = 1000
   Training settings: CPU: False Adam: True, Prec: fp16, Grad: True, TextTr: True EM: False, LR: 1.72e-06
 Allocated: 3.8GB
 Reserved: 3.9GB

Steps:   0%|                                                                                  | 0/1000 [00:00<?, ?it/s] Exception while training: 'Options' object has no attribute 'tag_drop_out'
 Allocated: 3.8GB
 Reserved: 3.9GB

Traceback (most recent call last):
  File "E:\NEWSD\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 981, in main
    for step, batch in enumerate(train_dataloader):
  File "E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\accelerate\data_loader.py", line 376, in __iter__
    current_batch = next(dataloader_iter)
  File "E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\torch\utils\data\dataloader.py", line 681, in __next__
    data = self._next_data()
  File "E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\torch\utils\data\dataloader.py", line 721, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "E:\NEWSD\stable-diffusion-webui\venv\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "E:\NEWSD\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\SuperDataset.py", line 168, in __getitem__
    example["instance_prompt"] = self.text_getter.create_text(instance_prompt,
  File "E:\NEWSD\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\finetune_utils.py", line 64, in create_text
    if shared.opts.tag_drop_out != 0:
  File "E:\NEWSD\stable-diffusion-webui\modules\shared.py", line 462, in __getattr__
    return super(Options, self).__getattribute__(item)
AttributeError: 'Options' object has no attribute 'tag_drop_out'
 CLEANUP:
 Allocated: 3.8GB
 Reserved: 3.9GB

 Cleanup completed.
 Allocated: 3.8GB
 Reserved: 3.9GB

 Cleanup Complete.
 Allocated: 3.8GB
 Reserved: 3.9GB

Steps:   0%|                                                                                  | 0/1000 [00:00<?, ?it/s]
 Training completed, reloading SD Model.
 Allocated: 0.0GB
 Reserved: 3.8GB

Memory output: {'Training completed, reloading SD Model.': '0.0/3.8GB'}
 Restored system models.
 Allocated: 2.0GB
 Reserved: 3.8GB

Returning result: Training finished. Total lifetime steps: 0
```

_Originally posted by @Merijeek in https://github.com/d8ahazard/sd_dreambooth_extension/issues/350#issuecomment-1329376455_

**Comment by zark119** [comment]
I still cannot create a new dreambooth ckpt. It worked before. The extention and the webui are updated.
------------------
Error completing request
Arguments: ('lm', 'v1-5-pruned.ckpt [a9263745]', 'euler-ancestral', 'runwayml/stable-diffusion-v1-5', '', False) {}
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 45, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 28, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\conversion.py", line 971, in extract_checkpoint
    pipe = pipe.to(map_location)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\diffusers\pipeline_utils.py", line 227, in to
    module.to(torch_device)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 927, in to
    return self._apply(convert)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 579, in _apply
    module._apply(fn)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 579, in _apply
    module._apply(fn)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 579, in _apply
    module._apply(fn)
  [Previous line repeated 2 more times]
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 602, in _apply
    param_applied = fn(param)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\torch\nn\modules\module.py", line 925, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
RuntimeError: CUDA out of memory. Tried to allocate 114.00 MiB (GPU 0; 12.00 GiB total capacity; 11.06 GiB already allocated; 0 bytes free; 11.28 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Traceback (most recent call last):
  File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 284, in run_predict
    output = await app.blocks.process_api(
  File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 983, in process_api
    data = self.postprocess_data(fn_index, result["prediction"], state)
  File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 913, in postprocess_data
    if predictions[i] is components._Keywords.FINISHED_ITERATING:
IndexError: tuple index out of range**

**Comment by d8ahazard** [comment]
Removing system-wide options from the base application without telling anybody? Welcome to the wild west of programming!

I am so irritated right now. Fixed with b041f9f
